examples/prescriptors/neat/prescribe.py

```diff
@@ -139,9 +129,10 @@ def prescribe(start_date_str: str,
             for geo in df['GeoID'].unique():
                 geo_pres = new_pres_df[new_pres_df['GeoID'] == geo]
                 geo_pred = new_pred_df[new_pred_df['GeoID'] == geo]
```

Comment:
Fix bug uncovered as results of fixing above bug: need to use concat instead of append for appending a vector to a matrix.